src/utils/create-utils.ts

```diff
@@ -403,12 +405,23 @@ type CreateImageRepositoryType = {
   namespace: string;
   isPrivate: boolean;
   bombinoUrl: string;
+  konfluxPublicInfo: ReturnType<typeof useKonfluxPublicInfo>;
 };
 
-export const createImageRepository = (
-  { application, component, namespace, isPrivate, bombinoUrl }: CreateImageRepositoryType,
+export const createImageRepository = async (
+  {
+    application,
+    component,
+    namespace,
+    isPrivate,
+    konfluxPublicInfo,
+  }: Omit<CreateImageRepositoryType, 'bombinoUrl'>,
   dryRun: boolean = false,
 ) => {
+  const [konfluxPublicInfoData, loaded, error] = konfluxPublicInfo;
+  const notifications =
+    loaded && !error ? konfluxPublicInfoData.integrations.image_controller.notifications ?? [] : []; // TBD
```

Comment:
create a hook that return notification data and pass that to createResource util.